src/Form/GraphQLQueryMapImportForm.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace Drupal\graphql\Form;
```

Comment:
Oh, now I see.. You built both! Nice. I don't think we need the other (manual) form for now. Can you split up this PR for just the upload form for now and submit the manual form separately? I need to think about that more.